R/make_instrument_auto.R

```diff
@@ -63,26 +63,44 @@ make_instrument_auto <- function(df, drop_which_when = FALSE,
     all(is.na(x) | x == "")
   })
 
-  # the rows that are not all missing
+  # the rows that are not all missing.
   if (drop_which_when == FALSE) {
+    # get the column number for the id and event name
```

Comment:
I'm not reading all this.
If you have a good test script, it will make sure that your code is correct.